service/domain/agreements.ts

```diff
@@ -1,73 +1,27 @@
 export class AgreementListItem {
   id: string | null;
   name: string;
-  description?: string;
-  creationTime?: Date;
-
-  constructor(
-    id: string | null,
-    name: string,
-    description?: string,
-    creationTime?: Date
-  ) {
-    this.id = id;
-    this.name = name;
-    this.description = description;
-    this.creationTime = creationTime;
-  }
+  description: string | null;
+  creationTime: Date | null;
 }
 
 export class AgreementVersion {
   id: string;
-  agreementId?: string;
+  agreementId?: string | null;
```

Comment:
Is this still optional or is it like the other optional -> null changes in this file?

```suggestion
  agreementId: string | null;
```

How about `texts`, are their contexts where it still might be optional?

Reply:
Good catch.  Fixed.

Reply:
Actually, seems like the build fails without these being both nullable and nullish.